develop/tutorials/articles/260-themes-and-layout-templates/07-embedding-portlets-in-themes-and-layout-templates.markdown

```diff
@@ -97,13 +106,85 @@ use wherever your theme is used.
 
 You successfully requested a portlet based on the entity and action types
 required, and created and deployed a module that retrieves the portlet and
-embeds it in your theme. Next, you'll learn a similar process to embed a portlet
-in your layout template.
+embeds it in your theme. 
+
+### Embedding a Portlet by Portlet Name [](id=embedding-a-portlet-by-portlet-name)
+
+If you'd like to embed a specific portlet in the theme, you can hardcode it by
+providing its instance ID and name:
+
+    <@liferay_portlet["runtime"]
+        portletProviderAction=portletProviderAction.VIEW
+        instanceId="INSTANCE_ID"
+        portletName="PORTLET_NAME"
+    />
+
+If your portlet is instanceable, an instance ID must be provided; otherwise, you
+can remove this line. To set your portlet to non-instanceable, set the property
+`com.liferay.portlet.instanceable` in the component annotation of your portlet
+to `false`. The portlet name must be the same as the property
+`javax.portlet.name`.
+ 
+Here's an example of an embedded portlet declaration that uses the portlet name 
+to embed a web content portlet:
+
+    <@liferay_portlet["runtime"]
+        portletName="com_liferay_journal_content_web_portlet_JournalContentPortlet"
+    />
+
+You can also set default preferences for an application. This process is covered 
+next.
+
+### Setting Default Preferences for an Embedded Portlet [](id=setting-default-preferences-for-an-embedded-portlet)
+
+Follow these steps to set default portlet preferences for an embedded portlet:
+
+1.  Temporarily assign the `VOID` variable to set portlet preferences using the 
+    `freeMarkerPortletPreferences` object as shown in the example below:
+
+        <#assign VOID = freeMarkerPortletPreferences.setValue(
+        "portletSetupPortletDecoratorId", "barebone") />
+    
+2.  Set the `defaultPreferences` attribute to use the 
+    `freeMarkerPortletPreferences` object you just configured:
+
+        <@liferay_portlet["runtime"]
+            defaultPreferences="${freeMarkerPortletPreferences}"
+            portletName="com_liferay_login_web_portlet_LoginPortlet"
+        />
+
+3.  Once the preferences have been set and passed to your portlet, you can reset
+    `freeMarkerPortletPreferences` object, so it can be fresh for the next 
+    portlet:
+
```

Comment:
> Once the preferences have been set and passed to your portlet, you should reset `freeMarkerPortletPreferences` object, so it can be fresh for the next portlet:

Use "should" instead of "can".